npm/ux/data.ts

```diff
@@ -10,7 +10,6 @@ export type ReData = {
   tfactory: any;
   errorBudget: any;
   logicalCounts: any;
-  new: boolean;
```

Comment:
Maybe isNew could be considered instead

Reply:
Not sure I follow. The 'new' property was only used when 'ensureSelected' was set, but one of your changes made that it never is, so I removed all the code related to 'ensureSelected', including the 'new' property.